# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmBCMemory.cpp
# Commit: 13ee9674ee35
# Full Hash: 13ee9674ee356acb36f3badb5ab2043d3f608eb1
# Author: Lars T Hansen <lhansen@mozilla.com>
# Date: 2021-10-18 16:09:29
# Regressor Bug: 1727084
# File Overlap Count: 1
# Description:
#   Bug 1727084 - Memory64 - Bulk memory operations. r=yury
#   
#   Implement all bulk memory operations on 64-bit memories.  Only
#   out-of-line paths are supported for memory.copy and memory.fill, for
#   now.
# ==============================================================================

diff -r b658cfe4b173 -r 13ee9674ee35 js/src/wasm/WasmBCMemory.cpp
--- a/js/src/wasm/WasmBCMemory.cpp	Mon Oct 18 09:58:14 2021 +0000
+++ b/js/src/wasm/WasmBCMemory.cpp	Mon Oct 18 09:58:14 2021 +0000
@@ -1711,7 +1711,7 @@
 //
 // Bulk memory.
 
-void BaseCompiler::emitMemCopyInline() {
+void BaseCompiler::memCopyInlineM32() {
   MOZ_ASSERT(MaxInlineMemoryCopyLength != 0);
 
   int32_t signedLength;
@@ -1916,7 +1916,7 @@
   freeI32(src);
 }
 
-void BaseCompiler::emitMemFillInline() {
+void BaseCompiler::memFillInlineM32() {
   MOZ_ASSERT(MaxInlineMemoryFillLength != 0);
 
   int32_t signedLength;